packages/blade/src/components/Form/FormLabel.tsx

```diff
@@ -158,9 +175,7 @@ const FormLabel = ({
       id={id}
       {...metaAttribute({ name: MetaConstants.FormLabel })}
     >
-      <BaseBox marginBottom={isLabelLeftPositioned ? 'spacing.0' : labelMarginBottom[size]}>
-        {textNode}
-      </BaseBox>
+      <BaseBox marginBottom={textNodeContainerMarginBottom()}>{textNode}</BaseBox>
```

Comment:
shouldn't the marginBottom be 0 always in case of labelLeftPositioned independent of ChipGroup? This will only push the text slightly upwards but won't necessarily align it right?